tests/test_expectation_maximization.py

```diff
@@ -90,7 +90,23 @@ def test_grid_SO3_uniform(test_ir, n_particles):
 def test_generate_xy_plane(test_ir, n_pix):
     """Test generation of xy plane."""
     xy_plane = test_ir.generate_xy_plane(n_pix)
-    assert xy_plane.shape == (n_pix**2, 3)
+    assert xy_plane.shape == (3, n_pix**2)
+
+    plane_2 = np.array([[-0.5, 0.5, -0.5, 0.5], [-0.5, -0.5, 0.5, 0.5], [0, 0, 0, 0]])
+
+    xy_plane = test_ir.generate_xy_plane(2)
+    assert np.allclose(xy_plane, plane_2)
+
+    plane_3 = np.array(
+        [
+            [-1, 0, 1, -1, 0, 1, -1, 0, 1],
+            [-1, -1, -1, 0, 0, 0, 1, 1, 1],
+            [0, 0, 0, 0, 0, 0, 0, 0, 0],
+        ]
+    )
+
+    xy_plane = test_ir.generate_xy_plane(3)
+    assert np.allclose(xy_plane, plane_3)
```

Comment:
I think this part of the test is a very good because it serves as readable documentation for the convention of `generate_xy_plane`. Good work.